code/auto_download/hospitalisations/README.md

```diff
@@ -0,0 +1,70 @@
+
+### European hospitalisation data pipeline
+
+#### Selecting data
+
+There is no single source covering hospitalisation data across all
+countries in the Forecast Hub. Data are also often heavily revised,
+particularly for recent dates, and can vary substantially between
+sources.
+
+We therefore create a combined set of what we identify as the most
+reliable data source, truncated to an appropriate date, for each
+country.
+
+-   The final dataset is saved as: [data-truth/ECDC/truth\_ECDC-Incident
+    Hospitalizations.csv](./data-truth/ECDC/truth_ECDC-Incident%20Hospitalizations.csv)
+-   The full pipeline is run with
+    [`hospitalisations-download.R`](./code/auto_download/hospitalisations-download.R)
+
+#### Data sources
+
+We also provide the full version of each data source:
+
+-   ECDC official:
+    [data-truth/ECDC/raw/official.csv](./data-truth/ECDC/raw/official.csv)
+    -   Source: [ECDC public
+        website](https://www.ecdc.europa.eu/en/publications-data/download-data-hospital-and-icu-admission-rates-and-current-occupancy-covid-19)
+    -   Origin: Mix of public data scraped from websites (typically
+        government data) by the ECDC, and data sent to ECDC directly
+        through the TESSy European data sharing agreement. All data are
+        processed and vetted by ECDC
+    -   Released: weekly on Thursdays. Each data point covers the
+        previous Monday-Sunday period. Data are incidence per 100,000
+        population.
+    -   Download with:
+        [`get-ecdc-official.R`](./code/auto_download/hospitalisations/get-ecdc-official.R)
+-   ECDC scraped:
```

Comment:
As it's not an official ECDC dataset perhaps we should remove ECDC here for clarify; could call it "EU scraped" perhaps?